detox/e2e/test/performance/login.e2e.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *******************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element testID when selecting an element. Create one if none.
+// *******************************************************************
+
+import fs from 'fs';
+
+import {Setup} from '@support/server_api';
+import client from '@support/server_api/client';
+import {
+    serverOneUrl,
+    siteOneUrl,
+} from '@support/test_config';
+import {
+    HomeScreen,
+    LoginScreen,
+    ServerScreen,
+} from '@support/ui/screen';
+import {computeStatsFromData} from '@support/utils/statistics';
+
+const N_RUNS = 10;
```

Comment:
This probably should go in an environment variable, among other things to get a higher number to calculate the baseline.

Reply:
That is more in the design part that in the code part (at least for now).

My idea is to recalculate every time there is a difference that we won't address. In other words:
- Whenever the performance improves
- Whenever the performance degrades but we decide it is a acceptable degradation
How will we do that? Mainly re-running the tests (maybe more iterations), getting the N, average and quasivariance, and updating the environment variables where this is taken from.